src/components/SHeaderItem.vue

```diff
@@ -1,38 +1,50 @@
 <script>
```

Comment:
Looks like you switched to two spaces as opposed to a tab elsewhere.

Reply:
converted all whitespace to tab with size of 4